pipelines/backup/cassandra-backup/Jenkinsfile

```diff
@@ -31,8 +31,20 @@ node() {
                 ansible_playbook_run(values)
             }
         }
+        currentBuild.result = 'SUCCESS'
     }
     catch (err) {
+        currentBuild.result = 'FAILURE'
+        slackSend (
+            channel: "${env.NOTIFY_SLACK_CHANNEL}",
+            color: 'danger',
+            message: "Build Failed - ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
+        )
         throw err
+    } finally {
+        step([$class: 'Mailer',
+            notifyEveryUnstableBuild: true,
+            recipients: "${env.NOTIFY_EMAIL_ADDRESS}",
```

Comment:
Same - global variable which isn't currently defined, please provide the value for this. Again, this will send all env alerts.